vault_program/src/set_is_paused.rs

```diff
@@ -0,0 +1,29 @@
+use jito_bytemuck::AccountDeserialize;
+use jito_jsm_core::loader::load_signer;
+use jito_vault_core::{config::Config, vault::Vault};
+use solana_program::{
+    account_info::AccountInfo, entrypoint::ProgramResult, program_error::ProgramError,
+    pubkey::Pubkey,
+};
+
+/// Processes the set `is_paused` instruction: [`crate::VaultInstruction::SetIsPaused`]
+pub fn process_set_is_paused(
```

Comment:
Slight update to this PR - but we talked to legal, and instdead of the Config ( Jito ) having the authority to pause, it should be up to the individual vault.

So, add a `pause_admin` to the vault, and that is who is allowed to pause/unpause the individual vault.